### ISSUE TITLE
Updating from inventorypets-1.7.10-1.4.0 to inventorypets-1.7.10-1.4.2 crashes dedicated server

### ISSUE THREAD

**Issue by Oxosorrel** [capnkirok/Inventory-Pets#62]
I just tried updating from 1.4 to 1.4.2 and my SMP server decided it didn't like the flavor of the update, sending this back to the chef for more secret sauce.

full crashlog:

http://pastebin.com/E9Y7X1JM

probably the important bits:

"Description: Exception in server tick loop

cpw.mods.fml.common.LoaderException: java.lang.ClassNotFoundException: com.inventorypets.InventoryPets
    at cpw.mods.fml.common.LoadController.transition(LoadController.java:163)
    at cpw.mods.fml.common.Loader.loadMods(Loader.java:544)
    at cpw.mods.fml.server.FMLServerHandler.beginServerLoading(FMLServerHandler.java:87)
    at cpw.mods.fml.common.FMLCommonHandler.onServerStart(FMLCommonHandler.java:314)
    at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:117)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:387)
    at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)"

**Comment by capnkirok** [comment]
Thanks, Oxosorrel for the report. I will take a look.

**Comment by capnkirok** [comment]
So, did you make any other changes to the server other than the Inventory Pets update? This error typically occurs when there's a client-only mod installed on the server (which could mean you have a mod that isn't supposed to be there, or a mod that has issues with the server or is conflicting). This could be IP or something else. Since I know IP 1.4.2 is working on servers, just want to know if there's anything else new or updated? Otherwise, we have to test one mod a time with IP and see where the problem is :/

**Comment by Oxosorrel** [comment]
I did update 

I try to keep my updates isolated to only a few at a time, since I run my own pack I'm all alone in for tech support. :)

When I moved from 1.4.0 to 1.4.2 I also moved from FTBlib and FTBUtil from 1.0.18.1 to 1.0.18.3. Only those 3 mods changed.

However when I encountered the above crash, I rolled Inventory Pets back to 1.4.0 my server restarted and ran just fine. 

Take a look at the mod list I have in my crash log, if any of those look suspicious let me know and I can try to do some isolation testing on my server tomorrow.
